snap/README.md

```diff
@@ -0,0 +1,99 @@
+# Kata Containers snap image
+
+* [Kata Containers snap image](#kata-containers-snap-image)
```

Comment:
Heh, I think that `Kata Containers snap image` link is redundant, as it just points to the title directly above it. Nuke if you get a chance?